src/Apps/FeatureAKG/Components/Feature.tsx

```diff
@@ -77,10 +78,10 @@ const Feature: React.FC<FeatureProps> = props => {
 
       {/* Video 1 */}
       <Media greaterThanOrEqual="md">
-        {video1?.large_src && <VideoSection src={video1.large_src} />}
+        {video1?.large_src && <ImageSection src={video1.large_src} />}
```

Comment:
These are going to be black and white gifs, so they should be small.

Reply:
You may want check what happens when this renders. Given that you're using `Media`, it's rendering both and hiding one. It's very likely both are being downloaded despite the breakpoint. If they're both small, maybe it's a non-issue. 

Might be time that we investigate adding support for the [picture element](https://www.w3schools.com/tags/att_source_srcset.asp)

Reply:
😮Gah.

I can try giving `picture` a spin tomorrow here. (This project has a tight deadline-- Wednesday, so I likely won't get to supporting it in palette by then).

Reply:
Build it in place and promote it once you've proved it out.